main/lsp/LSPTypechecker.h

```diff
@@ -124,7 +97,7 @@ class LSPTypechecker final {
     /**
      * Re-typechecks the provided files to re-produce error messages.
      */
-    TypecheckRun retypecheck(std::vector<core::FileRef> frefs, WorkerPool &workers) const;
+    std::vector<std::unique_ptr<core::Error>> retypecheck(std::vector<core::FileRef> frefs, WorkerPool &workers);
```

Comment:
This should still be `const`. The promise is: "`retypecheck` won't mutate any state".